src/components/Calculator.js

```diff
@@ -1,30 +1,67 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
```

Comment:
- [ ] [Optional] You don't need to import `operate` because you have already imported operate in `calculator.js`